mcc/capcom/capcom/capcom_test.go

```diff
@@ -230,8 +236,12 @@ func TestFindSecurityGroupsWithRange(t *testing.T) {
 			t.Error("Mismatched results and expectations length")
 		}
 		for k, v := range ret {
-			if v != tt.ret[k] {
-				t.Error("Unexpected output")
+			if v.String() != tt.ret[k].String() {
+				t.Errorf(
```

Comment:
I would have put this as subtest identifier and changed it to subtests instead of changing the error text. We can follow it up on a future PR.